micromagneticmodel/driver.py

```diff
@@ -325,3 +330,24 @@ def _setup_working_directory(system, dirname, mode, append=True):
         workingdir = system_dir / f"{mode}-{next_number}"
         workingdir.mkdir(parents=True)
         return workingdir
+
+    @staticmethod
+    def _conversion_to_hms(time_dfference):
+        total_seconds = int(time_dfference.total_seconds())
+        hours, remainder = divmod(total_seconds, 3600)
+        minutes, seconds = divmod(remainder, 60)
+        if time_dfference.total_seconds() < 1:
+            microseconds = time_dfference.microseconds
+            return f"{hours:02}:{minutes:02}:{seconds:02}.{microseconds:06}"
+        return f"{hours:02}:{minutes:02}:{seconds:02}"
+
+    def _update_info_json(self):
+        with open("info.json", encoding="utf-8") as jsonfile:
+            info = json.load(jsonfile)
+        info["end_time"] = self._end_time.isoformat(timespec="seconds")
+        info["elapsed_time"] = self._conversion_to_hms(
+            self._end_time - self._start_time
+        )
+        info["success"] = True
```

Comment:
I guess it could be useful to also indicate the failed status in here, i.e always update the info json after the simulation has finished (crashed). I'll propose changes in `drive`, please update this method accordingly.